.travis.yml

```diff
@@ -9,21 +9,22 @@ env:
   global:
     - CONTAINER: request-broker-web
     - APPLICATION_NAME: request_broker
-    - DJANGO_PORT: 8000
+    - APPLICATION_PORT: 80
```

Comment:
Should this be 8000? If not, can you explain?

Reply:
On the server, we want to deploy the application on port 80. Locally (in our container) we want it to run on port 8000.

When you run the local container, the environment variables in the Travis config file (and whatever is entered in the Travis UI) don't enter into the picture, so the values in the Compose file are used. When deploying, the Travis variables override everything else, which means that port 80 gets used for the deployment artifacts.

Reply:
Makes sense, thank!